tensorboard/program.py

```diff
@@ -122,6 +123,7 @@ def __init__(
         assets_zip_provider=None,
         server_class=None,
         subcommands=None,
+        experimental_plugins=[],
```

Comment:
Hmm; it would be great if we could avoid exposing this on `program`;
unlike `backend.application`, `program` is a public API and I’m not sure
that we want to commit to supporting this forever.

Do you have any thoughts about the best way to do this? The internal
clients that need this don’t call through `program` (instead calling
`TensorBoardWSGIApp` directly), but I see that `tensorboard.main` does.